development/index.md

```diff
@@ -0,0 +1,249 @@
+---
+layout: inner-page-no-masthead
+title: Scala development guarantees
+permalink: /development/
+includeTOC: true
+---
+
+## The TL;DR
+
+1. There are 2 distribution lines of Scala 3: **Scala Next** and **Scala LTS**.
+2. The **Scala Next** line is the **default** to be used by most users,
+   containing the latest features, bug fixes and improvements.
+3. The **Scala LTS** line is advised to be used for publishing **libraries**.
+   (Some especially conservative users might also choose it over Scala Next.)
+4. Bug fixes and usability improvements from the Scala Next line are regularly
+   backported to the currently supported LTS lines.
+5. Scala minor releases (3.x) are backwards compatible, meaning libraries built
+   with older versions can be consumed by newer versions. Patch releases (3.5.x)
+   are backwards and forwards compatible.
+6. Thus, existing source code continues to compile when upgrading to a new
+   version, except for rare situations when a bugfix requires breaking source
+   compatibility.
+
+## Contribute to Scala
+
+Scala is and always has been an open source effort. It is a collaboration
+between multiple entities (business, nonprofits, individuals) and relies on
+community contributions as much as the work of its maintainers.
+
+If you are interested to contribute code, be sure to start by visiting
+[the GitHub repository of the compiler](https://github.com/scala/scala3) (or the
+[standard library](https://github.com/scala/scala/tree/2.13.x/src/library)).
+
+If you’d like for your organization to work with us more closely, please contact
+scala.center (at) epfl.ch which will be able to arrange the possible
+collaboration.
```

Comment:
we need a link here, I think

Reply:
The lack of a link was suggested by @darjutak